capsules/src/ble_advertising_driver.rs

```diff
@@ -425,7 +327,7 @@ impl App {
                 }
                 ReturnCode::SUCCESS
             })
-            .unwrap_or_else(|| ReturnCode::EINVAL)
+            .unwrap_or(ReturnCode::FAIL)
```

Comment:
Changed all these `unwrap_or_else` because the closure is unused and is only if map fails i.e, if it's none